src/azure-cli/azure/cli/command_modules/appservice/static_sites.py

```diff
@@ -472,8 +473,8 @@ def _get_staticsite_location(client, static_site_name, resource_group_name):
                 if found_rg.lower() == resource_group_name.lower():
                     return static_site.location
 
-    raise CLIError("Static site was '{}' not found in subscription and resource group '{}'."
-                   .format(static_site_name, resource_group_name))
+    raise ResourceNotFoundError(f"Static site was '{static_site_name}' not found in subscription "
```

Comment:
what is the "f" I am seeing everywhere here?

Reply:
It's an [f-string](https://peps.python.org/pep-0498/). It's a feature that was added in python 3.6

F-strings are just a more concise way of formatting strings
You can do this: `f"first part of string {variable_name}"` instead of this: `f"first part of string {}".format(variable_name)`

Reply:
The lowest python version the CLI is run on is 3.6 so there won't be any python version issues